server/logrepl/replication.go

```diff
@@ -137,32 +138,69 @@ const maxConsecutiveFailures = 10
 
 var errShutdownRequested = errors.New("shutdown requested")
 
+type replicationState struct {
+	// conn is the current connection to the replica database, which can be re-established if it fails
```

Comment:
I'm assuming `conn` here in the comment is supposed to be `replicaConn`. Super small, nothing major